app/code/Magento/Swatches/view/frontend/web/js/swatch-renderer.js

```diff
@@ -217,7 +217,7 @@ define([
             jsonSwatchConfig: {},
 
             // selector of parental block of prices and swatches (need to know where to seek for price block)
-            selectorProduct: '.product-info-main',
+            selectorProduct: '.--product-info-main',
```

Comment:
Such selector is actually incorrect and violates https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-demarcation.html

Please add appropriate data attributes instead and add here.